legistar/events.py

```diff
@@ -197,7 +197,7 @@ def api_events(self, since_datetime=None):
                               item_key="EventId")
 
     def events(self, since_datetime=None):
-        for api_event in self.api_events(since_datetime):
```

Comment:
this shouldn't be necessary since since_datetime is the only arg for `api_events`?

I don't understand how this fixes things?

Reply:
We check kwargs, not args, for `since_datetime` to determine whether a scrape is partial: https://github.com/datamade/scrapers-us-municipal/blob/ca6d062c5d13226debfab83ba11e5db1440d46cf/lametro/events.py#L90

I prefer this explicit check to unpacking args, hence the change here rather than in the scraper.

Reply:
got it! Thanks, I didn't know we overrode api_events in the la metro scraper